src/platforms/zephyr/src/lib/platform_nifs.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * This file is part of AtomVM.
+ *
+ * Copyright 2019 Fred Dushin <fred@dushin.net>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
+ */
+
+#include <defaultatoms.h>
+#include <nifs.h>
+#include <platform_nifs.h>
+#include <term.h>
+
+// #define ENABLE_TRACE
+#include <trace.h>
+
+#include "zephyros_sys.h"
```

Comment:
Where is this file?

Reply:
in `AtomVM/src/platforms/zephyr/src/lib/`

Reply:
Is the file pending to be pushed?

Reply:
Yes, it was... I just forgot to add it to the commit before pushing the changes. It's there now.